app/views/partials/lib/commands/reviews/create/check.liquid

```diff
@@ -0,0 +1,8 @@
+{% liquid
+  assign c = '{ "errors": {}, "valid": true }' | parse_json
+
+  assign object = object | add_hash_key: 'valid', c.valid
```

Comment:
you might want to use new tag `hash_assign` which makes it easier to add keys to existing hash